server/services/interventionsService.test.ts

```diff
@@ -946,7 +946,7 @@ pactWith({ consumer: 'Interventions UI', provider: 'Interventions Service' }, pr
     },
     assignedTo: null,
     referenceNumber: 'HDJ2123F',
-    referral: {
+    referralFields: {
```

Comment:
this is a breaking API change. do we really need to do that?

Reply:
Ok, I've removed this rename.

Reply:
thanks!